spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/StreamListenerAnnotationBeanPostProcessor.java

```diff
@@ -60,7 +66,18 @@
 
 	private ConfigurableApplicationContext applicationContext;
 
-	public StreamListenerAnnotationBeanPostProcessor(DestinationResolver<MessageChannel> binderAwareChannelResolver, MessageHandlerMethodFactory messageHandlerMethodFactory) {
+	private final List<StreamListenerArgumentAdapter<?, Object>> streamListenerArgumentAdapters = new ArrayList<>();
+
+	private final List<StreamListenerResultAdapter<?, ?>> streamListenerResultAdapters = new ArrayList<>();
+
+	private int phase;
```

Comment:
The variables `phase`, `running` and `lifecycleMonitor` don't seem to be in use.